### PR TITLE
Improvement: Discord Rich Presence Glacite Tunnels

### PR DESCRIPTION
**Pull request by NetheriteMiner**
## What
Adds new images to rich presence, including the new glacite tunnels area and some previous (but significant) areas whose images were not unique.

Jerry's Workshop and the Glacite Mineshaft were untested, but everything should work.

The below images do not include all the new images added, but they do include some of the best ones.
<details>
<summary>Images</summary>

glacial caves
dungeon
garden
magma fields

</details>

## Changelog Improvements
+ New Images for Discord Rich Presence. - NetheriteMiner
    * Updated important areas to have new images.

### CODE REVIEW

File: src/main/java/at/hannibal2/skyhanni/features/misc/discordrpc/DiscordLocationKey.kt [hunk 1]
```diff
@@ -55,14 +75,29 @@ class DiscordLocationKey {
         "Catacombs Entrance" to "coal-mine",
         "Colosseum Arena" to "colosseum",
         "Museum" to "unincorporated",
-        "Mines of Divan" to "gold-mine",
```
Comment (lunaynx) [inline src/main/java/at/hannibal2/skyhanni/features/misc/discordrpc/DiscordLocationKey.kt:58]: Where did this go?

Comment (Rohan-Sangani) [reply src/main/java/at/hannibal2/skyhanni/features/misc/discordrpc/DiscordLocationKey.kt:58]: Great question! (fixed in 6c0053a)

**Review by hannibal002 — APPROVED** [review]
worked fine
image

**Comment by lineargraph** [comment]
Maybe the source images should be in the repo too somewhere. Right now they are only on discord, which makes tracking them kind of hard (as well as would be doing forks or new updates to the images). Also, boo, AI!

**Pull request merged**